spec/info/description.md

```diff
@@ -122,7 +129,16 @@ Example:
 
 You can generate your token on [API console](https://app.zenvia.com/home/api) inside Zenvia platform website.
 
+
 ## JWT
+
 The JWT token is primarily used by front-end applications for user interactions.
 
 For server to server integrations use [TOKEN](/#section/Authentication/TOKEN) authentication type.
+
+
+# SDKs
+
+SDKs make it easy to use our APIs using a programming language including in your software.
```

Comment:
O que você no final com "including in your software"? Eu não entendi. Seria do tipo incluir o SDK no software da pessoa?

Reply:
Sim.

Reply:
Então acho que não ficou legal. Seguindo a linha do que eu entendi que você quis escrever eu usaria:
SDKs make it easy to use our APIs just by including them in your software.
Ou ainda:
SDKs make it easy to use our APIs by using them while developing your software.